templating/templating.go

```diff
@@ -64,12 +78,198 @@ func Populate(name, description string, events []Event) (desc string, err error)
 		"stringsSplit":      strings.Split,
 	}
 
+	sprigFuncMap := FilterKeys(sprig.FuncMap(), []string {
```

Comment:
better call FilterKeys only once for start, not with call "Populate"

Reply:
I moved out this and funcsMap out of "Populate" scope